docs/index.rst

```diff
@@ -0,0 +1,87 @@
+.. meta::
+   :description: Omniperf documentation and reference
+   :keywords: Omniperf, ROCm, profiler, tool, Instinct, accelerator, AMD
+
+********
+Omniperf
+********
+
+Omniperf documentation provides a detailed breakdown of all facets of Omniperf.
+In addition to a full deployment guide with installation instructions, this
+documentation also explains the ideas motivating the design behind the tool and
+its components.
+
+Omniperf is open source and hosted at `<https://github.com/ROCm/omniperf>`__.
```

Comment:
This paragraph should be below the next paragraph

Reply:
updated